pkg/apis/bus/v1alpha1/events.go

```diff
@@ -30,6 +30,12 @@ const (
 	// PodEvictedEvent is triggered if Pod was deleted
 	PodEvictedEvent Event = "PodEvicted"
 
+	// PodPendingEvent is triggered if Pod was pending
+	PodPendingEvent Event = "PodPending"
+
+	// PodRunningEvent is triggered if Pod was running
+	PodRunningEvent Event = "PodRunning"
```

Comment:
What's the intention for PodRunning?

Reply:
When the pod corresponding to a podFailed/podEvicted event returns to the running state, its delayed action should be canceled. So we need to be able to detect the pod running event.

Reply:
It makes sense:)